tests/bluetooth/controller/common/include/helper_pdu.h

```diff
@@ -57,7 +57,7 @@ void helper_pdu_verify_ping_rsp(const char *file, uint32_t line, struct pdu_data
 
 void helper_pdu_verify_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
 				   void *param);
-void helper_pdu_verify_slave_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
+void helper_pdu_verify_peripheral_feature_req(const char *file, uint32_t line, struct pdu_data *pdu,
```

Comment:
Keep in mind that by using `peripheral` there likely need to be some re-indendation and line breaks.

Reply:
There were, and I think I covered them all

Reply:
This seems to be above the recommended (but not strict) 80 char line, and the `void *param` on next line is incorrectly indented :) Not sure if there's more like this

Reply:
80 is recommended, but the max is 100

Reply:
I see now that I missed the indentation; fixed.
Checked other places for correct indentation as well